Examples/Scripts/Python/truth_jet_test.py

```
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)